### ISSUE TITLE
paper comments target audience and napari

### ISSUE THREAD

**Issue by pr4deepr** [lys-devel/lys#13]
Hi @lys-devel 
    
Great software.. 

In the paper, the **target audience hasn't been stated clearly**. 
I know you mention its low code, but it really depends on your target audience. 
Many disciplines have multidimensional data, so it would be great to give a general idea of the target audience. 
This may be a low code solution to physicists with coding experience, but to a person with a background in biology and intermediate coding experience, this is more than just 'low code'. 

I think you do need to have some knowledge of the respective libraries and use cases. 

Have you seen [napari](https://napari.org/stable/)? It is worth mentioning it in your paper and how different lys is. You can run IPython console in it and visualize it at the same time. Its mostly aimed for visualization but there is an ecosystem of [plugins](https://www.napari-hub.org/).  It can also handle large dask arrays and xarrays).

Cheers
Pradeep

**Comment by pr4deepr** [comment]
https://github.com/openjournals/joss-reviews/issues/5869

**Comment by lys-devel** [comment]
Thank you for the fruitful comment. 

I have tried `napari` as you suggested. After understanding what other similar software can do, I noticed that `lys` is a general platform for data analysis and visualization (not a simple image viewer/analysis software).
Therefore I added the paragraph that describes above point and spcify target audience as in line 51-56 in the [new manuscript](https://github.com/openjournals/joss-papers/blob/joss.05869/joss.05869/10.21105.joss.05869.pdf). In addition, I also compare `lys` with other similar software including `napari` in line 101-113.

I hope these modifications meet the publication criteria in JOSS.
